var/spack/repos/builtin/packages/aluminum/package.py

```diff
@@ -57,8 +61,12 @@ def cmake_args(self):
                 '-DALUMINUM_ENABLE_MPI_CUDA_RMA:BOOL=%s' %
                 ('+cuda_rma' in spec)])
         else:
+            args.extend(
+                '-DALUMINUM_ENABLE_MPI_CUDA:BOOL=%s' % ('+ht' in spec))
+
+        if '@:0.1,0.6.0:':
             args.extend([
-                '-DALUMINUM_ENABLE_MPI_CUDA:BOOL=%s' % ('+ht' in spec)])
+                '-DCUB_DIR:FILEPATH=%s' % spec['cub'].prefix.include])
```

Comment:
`list.extend` takes a list, `list.append` takes a single argument. I think you're using both wrong in multiple places in this package.

Reply:
Fixed.